torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -989,22 +992,29 @@ class IrParser {
                 unaryOp(UnaryOpType::Reciprocal, num_features);
             auto* grad_in = mul(mul(reciprocal_size, rstd), inner);
 
-            value_map.emplace(node->output(0)->unique(), grad_in);
-
-            // TODO: grad_bias and grad_weight are disabled because
-            // they are incompabilble with grad_in fusion
-            // Requires seperate kernels
+            if (output_mask[0]) {
```

Comment:
Seems like you're inserting dummy 0-dim tensors in place when there aren't certain outputs required. Is that true? Is this the dangling output you're referring to for segmentation?

Reply:
Yes. This is the portion that's causing earlier segfault during the execution of segmented fusion. Now it should be working fine.